.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 *.swp
 build/
+vendor/
```

Comment:
So once you switch to go modules, there is no need for this. The `go.mod` file guards the versions that are your dependencies. In case some dependency is not using semantic versioning, you may refer to a particular commit-SHA (still, so crazy devs can `push -f` to master).

In case some deps go crazy (some editors _help_ you update `go.mod`), you can just restore the single file with the past versions and run `go mod vendor`. This gives you a lot of safety.

Of course having it committed is a possibility, but it feels still very pre-v1.10 :)

Reply:
I would suggest also to switch slowly to golang 1.13 or even 1.14. The migration should be unproblematic. You may use the testing matrix to test multiple versions (provided the minutes-allowance for GH-actions can handle it). Example: https://gist.github.com/vikin91/9239b6db98660415369968cb778e87d5

Reply:
Thanks for the explanation. Sounds very reasonable - I will have a closer look into this. 
I will address golang upgrade and dep management in a different PR.